modules/openapi-generator/src/main/resources/rust/reqwest/configuration.mustache

```diff
@@ -22,6 +29,44 @@ pub struct ApiKey {
     pub key: String,
 }
 
+{{#withAWSV4Signature}}
+#[derive(Debug, Clone)]
+pub struct AWSv4Key {
+    pub access_key: String,
+    pub secret_key: String,
```

Comment:
shouldn't this be wrapped in something like [secrecy](https://crates.io/crates/secrecy)?

Reply:
Didn't knew about secrecy, I will fix this

Reply:
FWIW I don't insist on secrecy specifically (just happens to be the one I've used).